packages/databyss-ui/components/Editor/slate/markup.js

```diff
@@ -0,0 +1,78 @@
+import cloneDeep from 'clone-deep'
+import { Editor, Value } from 'slate'
+
+export const stateToSlate = (slate, _id) => {
```

Comment:
I think you have these named backwards...this one should be `slateToState`, as in, `slate block` => `state entity`. Maybe a better name would be slateBlockToStateEntity